### ISSUE TITLE
CPO doxygen warnings

### ISSUE THREAD

**Issue by tjhei** [geodynamics/aspect#4995]
```
/ssd/aspect-git-clean/include/aspect/particle/property/crystal_preferred_orientation.h:181: warning: argument 'volume_fractions' of command @param is not found in the argument list of aspect::Particle::Property::CrystalPreferredOrientation< dim >::compute_derivatives(const unsigned int cpo_index, const ArrayView< double > &data, const unsigned int mineral_i, const SymmetricTensor< 2, 3 > &strain_rate, const Tensor< 2, 3 > &velocity_gradient_tensor, const std::array< double, 4 > &ref_resolved_shear_stress) const
/ssd/aspect-git-clean/include/aspect/particle/property/crystal_preferred_orientation.h:181: warning: argument 'rotation_matrices' of command @param is not found in the argument list of aspect::Particle::Property::CrystalPreferredOrientation< dim >::compute_derivatives(const unsigned int cpo_index, const ArrayView< double > &data, const unsigned int mineral_i, const SymmetricTensor< 2, 3 > &strain_rate, const Tensor< 2, 3 > &velocity_gradient_tensor, const std::array< double, 4 > &ref_resolved_shear_stress) const
/ssd/aspect-git-clean/include/aspect/particle/property/crystal_preferred_orientation.h:181: warning: argument 'volume_fraction_mineral' of command @param is not found in the argument list of aspect::Particle::Property::CrystalPreferredOrientation< dim >::compute_derivatives(const unsigned int cpo_index, const ArrayView< double > &data, const unsigned int mineral_i, const SymmetricTensor< 2, 3 > &strain_rate, const Tensor< 2, 3 > &velocity_gradient_tensor, const std::array< double, 4 > &ref_resolved_shear_stress) const
/ssd/aspect-git-clean/include/aspect/particle/property/crystal_preferred_orientation.h:193: warning: The following parameters of aspect::Particle::Property::CrystalPreferredOrientation::compute_derivatives(const unsigned int cpo_index, const ArrayView< double > &data, const unsigned int mineral_i, const SymmetricTensor< 2, 3 > &strain_rate, const Tensor< 2, 3 > &velocity_gradient_tensor, const std::array< double, 4 > &ref_resolved_shear_stress) const are not documented:
  parameter 'cpo_index'
  parameter 'data'
  parameter 'mineral_i'
/ssd/aspect-git-clean/include/aspect/particle/property/crystal_preferred_orientation.h:358: warning: The following parameter of aspect::Particle::Property::CrystalPreferredOrientation::set_rotation_matrix_grains(const unsigned int cpo_data_position, const ArrayView< double > &data, const unsigned int mineral_i, const unsigned int grain_i, const Tensor< 2, 3 > rotation_matrix) const is not documented:
  parameter 'rotation_matrix'
```
@MFraters Can you please take a look?

**Comment by MFraters** [comment]
ah, I see the issue. I will make a pull request to fix it.

**Comment by gassmoeller** [comment]
fixed by #4997
